object_detection/g3doc/running_pets.md

```diff
@@ -76,7 +76,7 @@ tar -xvf annotations.tar.gz
 tar -xvf images.tar.gz
 python object_detection/create_pet_tf_record.py \
     --label_map_path=object_detection/data/pet_label_map.pbtxt \
-    --data_dir=`pwd` \
```

Comment:
Looking closer, this `pwd` is the correct value for this flag. We want the --data_dir path to point to the directory where {images/annotations}.tar.gz are unzipped. If you copied and pasted this code block, it would unzip them into the root of the git checkout, not object_detection/data/.